cot/tests/admin.rs

```diff
@@ -0,0 +1,158 @@
+use std::error::Error;
+
+use async_trait::async_trait;
+use cot::admin::AdminApp;
+use cot::auth::db::{DatabaseUser, DatabaseUserApp};
+use cot::cli::CliMetadata;
+use cot::config::{
+    AuthBackendConfig, DatabaseConfig, MiddlewareConfig, ProjectConfig, SessionMiddlewareConfig,
+};
+use cot::middleware::{AuthMiddleware, SessionMiddleware};
+use cot::project::{MiddlewareContext, RegisterAppsContext};
+use cot::static_files::StaticFilesMiddleware;
+use cot::test::{TestServer, TestServerRunning};
+use cot::{App, AppBuilder, BoxedHandler, Project, ProjectContext};
+use fantoccini::{Client, ClientBuilder, Locator};
+
+struct HelloApp;
+
+const DEFAULT_USERNAME: &str = "admin";
+const DEFAULT_PASSWORD: &str = "admin";
```

Comment:
nit: it's just confusing a bit IMHO
```suggestion
const DEFAULT_USERNAME: &str = "admin";
const DEFAULT_PASSWORD: &str = "admin";

struct HelloApp;
```